pkgs/development/python-modules/sphinxcontrib-autoapi/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, astroid
+, jinja2
+, sphinx
+, pyyaml
+, unidecode
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autoapi";
+  version = "1.5.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hq96s0kg5v6zkb813k4w11rx0n1hv9iqyllsj24sa0qfsgbdzwm";
+  };
+
+  propagatedBuildInputs = [ astroid jinja2 pyyaml sphinx unidecode ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Thx @jonringer, I've added the tests.